kadane_algorithm/kadane.cpp

```diff
@@ -0,0 +1,34 @@
+#include <array>
```

Comment:
Provide more information
in the `@details` part of the block.

Confused? See #962 as a reference. 🙂
```suggestion
/**
 * @file
 * @brief Implementation of [Kadane Algorithm](https://en.wikipedia.org/wiki/Kadane%27s_algorithm)
 *
 * @details
 * <full-description here>
 *
 * @author [Ayush Singh](https://github.com/ayush523)
 */
#include <array>
```